src/labone/core/subscription.py

```diff
@@ -87,11 +87,20 @@ def __init__(
         *,
         path: LabOneNodePath,
         register_function: t.Callable[[weakref.ReferenceType[DataQueue]], None],
+        value_packaging_function: t.Callable[[AnnotatedValue], AnnotatedValue]
```

Comment:
issue: the `value_packaging_function` must not be part of the `DataQueue` but the `StreamingHandle`. Otherwise, it will be called n times for n forks.